src/sentry/utils/security/orgauthtoken_token.py

```diff
@@ -2,15 +2,23 @@
 from base64 import b64decode, b64encode
 from datetime import datetime
 
-from django.conf import settings
-
+from sentry import options
 from sentry.utils import hashlib, json
 
 SENTRY_ORG_AUTH_TOKEN_PREFIX = "sntrys_"
 
 
+class SystemUrlPrefixMissingException(Exception):
+    "system.url-prefix is not set. You need to set this to generate a token."
```

Comment:
Is this the error message? Are you sure this is correct?

Reply:
No, I _think_ this is just a comment (?) but TBH I took this from other custom exception examples - this is not really accessible on the exception being raised. I am also OK with removing this - not sure..?

Reply:
A comment is either `#` for single line or `"""` for docs. I might be wrong, maybe the syntax was updated or the parser is just very flexible and treats your code as valid.

If you can, just use `#`.
```suggestion
    # system.url-prefix is not set. You need to set this to generate a token.
```